src/server.js

```diff
@@ -60,7 +52,9 @@ server.FullServer = class extends BaseServer {
 
     if (!clientSocketPort && !clientWebSocketPort) {
       throw Error('Please specify either clientSocketPort, clientWebSocketPort, or both.')
-    } else {
+    } 
+    else {
```

Comment:
It looks like you prefer Stroustrup style of curly bracing. I just looked it up, and eslint supports it.

https://eslint.org/docs/rules/brace-style

I'll add it to `.eslintrc` and then we can consistently encourage it / automatically convert it everywhere.

Reply:
Thanks, I think it makes the code more readable.